### ISSUE TITLE
interstitial sizes

### ISSUE THREAD

**Issue by ageller** [ageller/EmeryInteractive#16]
(ideally) Figure out some way to show how to calculate their sizes based on their packing/positions

**Comment by ageller** [comment]
given the positions, we know that r1 (big) + r2 (interstitial) = distance between positions, but need nearest neighbors

**Comment by emeryjdk** [comment]
I think we should plot coordination for only one of the interstitial sites. Here they are for the following (dashes separate atomic positions):

**Simple Cubic**:
- [x] Cubic site (0.5-0.5-0.5) is coordinate with 8 atoms atoms at:
0-0-0, 1-0-0, 0-1-0, 0-0-1, 1-1-0 1-0-1, 0-1-1, 1-1-1.

**FCC**:
- [x] Octahedral site (0.5-0.5-0.5) is coordinate with 6 atoms atoms at:
0.5-0.5-0, 0.5-0.5-1, 0.5-0-0.5, 0.5-1-0.5, 0-0.5-0.5, 1-0.5-0.5. 
- [x] Tetrahedral site (0.25-0.25-0.25) is coordinate with 4 atoms atoms at:
0-0-0, 0.5-0.5-0, 0.5-0-0.5, 0-0.5-0.5

**BCC** (this structure has distorted sites. "Octehedral" sites actually only touch 2 atoms, but we typically show 6:
- [x] Octahedral site (0.5-0-0.5) is coordinate with 6 atoms atoms at:
0-0-0, 1-0-0, 0-0-1, 1-0-1, 0.5-0.5-0.5, 0.5-(-0.5(negative! in the adjacent unit cell))-0.5. 
- [x] Tetrahedral site (0.25-0.25-0.5) is coordinate with 4 atoms atoms at:
0-0-0, 0.5-0.5-0.5, 0-0-1, 0.5-(-0.5(negative! in the adjacent unit cell))-0.5.

@ageller

**Comment by ageller** [comment]
@emeryjdk 
I added the cubic site coordination:  Is that how you intended it to look? 

But now I'm a little bit confused about how the coordinations are meant to work for the others. 

1. Do you want to show both coordinations (one for each interstitial site) at the same time, or should that be controlled somehow with the buttons on the left panel?
2. For the FCC, we have a coordination that includes only atoms (right?).  How does this relate to what you've shown above?  Should we have an atoms-only coordination for BCC and SC?

**Comment by emeryjdk** [comment]
@ageller 

Yes - the cubic coordination looks great, but I think we'll need a radio button when under the coordination view. We'll need to see atomic coordination and interstitial coordination. So:

1. Yes, we should have a button on the panel that controls which coordination we're viewing (atomic, tetrahedral, octahedral).
2. In FCC we have coordinations for atom-to-atom and interstitial (tet, oct) to atoms. So, we need a button for each that we can use to toggle views.

Does that make sense? I can draw pictures if that would help.

**Comment by ageller** [comment]
@emeryjdk 

I added the coordinations that you provided above, and radio buttons within the main Coordination to define which to show.  Related to that I have a question:

Is it correct that there are no atomic coordinations to plot for SC and BCC?  (And if so, I will remove the related radio buttons). 

And also, please let me know if anything looks wrong in the coordinations that I added.
